src/client/components/MyProfile/YourUploadedCVs/RatingStars.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Star } from './Star';
+
+export function RatingStars({ averageStars }) {
+  const reminder = `${((averageStars % 20) / 20) * 100}%`;
+  const filledStar = parseInt(averageStars / 20, 10);
+  const stars = [1, 2, 3, 4, 5];
+
+  return (
+    <div style={{ display: 'flex', direction: 'row' }}>
+      {stars.map((x, i) => {
+        return i < filledStar ? (
+          <div style={{ marginLeft: '8px' }}>
+            <Star averageRatingInPercent="100%" />{' '}
+          </div>
+        ) : (
+          <div style={{ marginLeft: '8px' }}>
+            <Star
+              averageRatingInPercent={
+                i < filledStar + 1 ? reminder.toString() : '0'
+              }
+            />
```

Comment:
This is tricky for me to understand. 

I would expect to render either a filled or an empty star, but so much more is going on. THe return `i < filledStar`, the `averageRatingInPercent`, this one is super tricky: `i < filledStar + 1 ? reminder.toString() : '0'`

Try simplify this! Make it as easy for another developer to read your code as is possible!

Reply:
Here we have 5 star so it maps from the star array when filled star meets so the stars comes with the styale of filles stayles otherwise they come with the reminder stars style. for example if the average star is 3.75 comes 3 stars with the style of width=100% and 2 stars comes with the stayle of reminder amount for width. however it is not correct and should comes just one star with the style of width 0.75. that is why I made the solution to come just one star after the filled stars by this line  averageRatingInPercent={
                i < filledStar + 1 ? reminder.toString() : '0'
              }. 

I know this is a little bit tricky and maybe complex. Buty I could not find a better way to meet all things togethher. for getting the resault this code just need averageStarInPercentage.